src/components/Toast.tsx

```diff
@@ -94,13 +94,7 @@ export const Toast = ({
           slotTitleLeft={slotTitleLeft}
           slotTitleRight={slotTitleRight}
           slotDescription={slotDescription}
-          slotAction={
-            actionDescription && (
-              <$Action asChild altText={actionAltText}>
-                {slotAction}
-              </$Action>
-            )
-          }
+          slotAction={slotAction}
```

Comment:
Figure out if we can re-introduce `$Action`

Reply:
Add it it back! Appearantly `altText` is required for Action to render.